formats/json/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -295,22 +442,12 @@ public class JsonBuilder internal constructor(json: Json) {
      */
     public var coerceInputValues: Boolean = json.configuration.coerceInputValues
 
-    /**
-     * Switches polymorphic serialization to the default array format.
-     * This is an option for legacy JSON format and should not be generally used.
-     * `false` by default.
-     *
-     * This option can only be used if [classDiscriminatorMode] in a default [ClassDiscriminatorMode.POLYMORPHIC] state.
-     */
-    public var useArrayPolymorphism: Boolean = json.configuration.useArrayPolymorphism
-
     /**
      * Name of the class descriptor property for polymorphic serialization.
-     * "type" by default.
+     * `"type"` by default.
```

Comment:
If you want to use code-like formatting, then I think we should omit quotes, since value is not quoted:

```suggestion
     * `type` by default.
```